sdk/communication/azure-communication-chat/azure/communication/chat/_shared/response.py

```diff
@@ -0,0 +1,8 @@
+# ------------------------------------------------------------------------
```

Comment:
the _shared folder is shared between all service SDKs, like SMS SDK has the same folder and same files/methods. Which means adding one method/module to this folder, other teams must do the same. 
So I suggest:
1. Talk to Tural to ask if above is the same case, if yes, ask if this will become a common method that other teams will use, if yes, then your PR is good
2. If this module and method is only for Chat team, suggest move it to azure/communication/chat/_util.py after talked to Tural.

Reply:
Thanks for the feedback. 
Currently, I don't commit the test files in order to reduce the unnecessary review workload. Yes, I will run and upload new test files after review.  

For the helper file in shared folder, what do you think? @turalf

Reply:
I guess this file is specific to the chat module. If that is the case please keep this file outside of the `_shared`  folder. 
Also it seems that this method is used only in 1 file. If you are not planning to use it somewhere else in the future, better to keep in that file.